lnst/RecipeCommon/Perf/Evaluators/BaselineFlowAverageEvaluator.py

```diff
@@ -76,13 +76,19 @@ def _average_diff_comparison(
         baseline: SequentialPerfResult,
     ):
         difference = result_averages_difference(target, baseline)
-        result_text = "New {name} is {diff:.2f}% {direction} from the baseline {name}".format(
+        result_text = "New {name} average is {diff:.2f}% {direction} from the baseline {name}".format(
             name=name,
             diff=abs(difference),
             direction="higher" if difference >= 0 else "lower",
         )
-        comparison = abs(difference) <= self._pass_difference
-        if comparison:  # TODO
+
+        cpu = "_cpu_" in name
+
+        #  (                 flow metrics                 ) or (                cpu metrics                )
+        if (not cpu and difference > self._pass_difference) or (cpu and difference < -self._pass_difference):
+            comparison = ResultType.WARNING
+            result_text = "IMPROVEMENT: " + result_text
+        elif (not cpu and difference >= -self._pass_difference) or (cpu and difference <= self._pass_difference):
```

Comment:
looking at the logs from the test jobs, i'm thinking if maybe it would make sense to add "PASS:" and "FAIL:" to the individual comparison lines same as for "IMPROVEMENT:" too?

e.g.
```
        Configured 4.34% difference as acceptable
        PASS: New generator_results average is 1.65% lower from the baseline generator_results
        PASS: New generator_cpu_stats average is 0.02% higher from the baseline generator_cpu_stats
        FAIL: New receiver_results average is 4.36% lower from the baseline receiver_results
        PASS: New receiver_cpu_stats average is 3.47% lower from the baseline receiver_cpu_stats
```

what do you think @enhaut @jtluka

Reply:
nice idea, thanks

Reply:
i've replaces `success` property with `result`. now it should be consistent across project